### DISCUSSION TITLE
Generate  python code

### DISCUSSION THREAD

**Discussion by Yoris95** [sinaptik-ai/pandas-ai#1262] [Q&A]
I am a newbie using pandasai. Can you show me how to show or generate last python code after we run the agent? My code snippet looks like this:

from pandasai import Agent

pandas_ai = Agent(df, memory_size=10,
                  description="You are a data analysis agent. Your main goal is to help non-technical users to analyze data.",
                  config={"llm": llm,
                          "enforce_privacy": True,
                          "response_parser": StreamlitResponse,
                          "save_charts": True,
                          "enable_cache": True,
                          "verbose": False})

**Comment by affanali2k3** [comment]
print(pandas_ai.last_code_executed) // Stores the last executed python code.
